### PR TITLE
Added support for conferencing_profiles in userinfo endpoint

### PR DESCRIPTION
**Pull request by lavy-blaggan**
As described at https://docs.cronofy.com/developers/api/conferencing-services/conferencing-profiles/

### CODE REVIEW

File: src/Cronofy/UserInfo.cs [hunk 1]
```diff
@@ -225,85 +232,189 @@ public override string ToString()
...
+          /// <param name="other">
+          /// The <see cref="Cronofy.UserInfo.ConferencingProfile"/> to compare with the current
+          /// <see cref="Cronofy.UserInfo.ConferencingProfile"/>.
+          /// </param>
+          /// <returns>
+          /// <c>true</c> if the specified <see cref="Cronofy.UserInfo.ConferencingProfile"/> is equal
+          /// to the current <see cref="Cronofy.UserInfo.ConferencingProfile"/>; otherwise,
+          /// <c>false</c>.
+          /// </returns>
+          public bool Equals(Profile other)
```
Comment (CronofyMatt) [inline src/Cronofy/UserInfo.cs:314]:
```suggestion
          public bool Equals(ConferencingProfile other)
```

I believe theres a possible infinite loop  when calling `Equals(object obj)` as this parameter needs to be the type of the class itself.

**Review by CronofyMatt — CHANGES_REQUESTED** [review]
Hey @lavy-blaggan,

Thank you for your contribution!

I have taken a moment to review this and it seems that its very nearly there in terms of functionality. Theres just a couple of issues that would need some attention before we could accept this into the SDK.

- It seems theres a possible infinite loop in the `Equals` method of the `ConferencingProfile` class.
- Theres a couple of CI warnings around invalid `CREF` tags in some of the doc comments. I think these need the namespace/parent class prefixed onto them to point to the right thing.

Once these issues are resolved we would be happy to accept your contribution.

All the best,
Matt

File: src/Cronofy/UserInfo.cs (hunk 1 above)
Comment (lavy-blaggan) [reply src/Cronofy/UserInfo.cs]: Great pick up.

**Comment by lavy-blaggan** [comment]
> Hey @lavy-blaggan,
> 
> Thank you for your contribution!
> 
> I have taken a moment to review this and it seems that its very nearly there in terms of functionality. Theres just a couple of issues that would need some attention before we could accept this into the SDK.
> 
> * It seems theres a possible infinite loop in the `Equals` method of the `ConferencingProfile` class.
> * Theres a couple of CI warnings around invalid `CREF` tags in some of the doc comments. I think these need the namespace/parent class prefixed onto them to point to the right thing.
> 
> Once these issues are resolved we would be happy to accept your contribution.
> 
> All the best, Matt

Thank you for feedback, @CronofyMatt.

I have resolved the issues. Let me know if there's anything else.

Cheers,
Lavy

**Review by ja-sukhman — APPROVED** [review]

**Comment by CronofyMatt** [comment]
Hi @lavy-blaggan,

Thanks for getting back to us and making the requested changes. I have given everything another look over and tested out the functionality and can see its working as expected. 

I will get this merged now and get it deployed to NuGet.

Thank you again for your contribution!

All the best,
Matt

**Review by CronofyMatt — APPROVED** [review]

**Pull request merged**